README.md

```diff
@@ -1,15 +1,39 @@
 # Mink WebDriver Classic Driver
 
+[![PHP Version](http://poser.pugx.org/mink/webdriver-classic-driver/require/php)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Latest Stable Version](https://poser.pugx.org/mink/webdriver-classic-driver/v)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Latest Unstable Version](https://poser.pugx.org/mink/webdriver-classic-driver/v/unstable)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Total Downloads](https://poser.pugx.org/mink/webdriver-classic-driver/downloads)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![CI](https://github.com/minkphp/webdriver-classic-driver/actions/workflows/ci.yml/badge.svg)](https://github.com/minkphp/webdriver-classic-driver/actions/workflows/ci.yml)
 [![License](https://poser.pugx.org/mink/webdriver-classic-driver/license)](https://github.com/minkphp/webdriver-classic-driver/blob/main/LICENSE)
 [![codecov](https://codecov.io/gh/minkphp/webdriver-classic-driver/branch/main/graph/badge.svg?token=11hgqXqod9)](https://codecov.io/gh/minkphp/webdriver-classic-driver)
 
-## Usage Example
+A [php-webdriver](https://github.com/php-webdriver/php-webdriver)-based driver
+for [Mink](https://github.com/minkphp/Mink), supporting Selenium 2-4 and tested on Chrome, Firefox and Edge.
 
-``` php
+## Installation
+
+### For Mink/Standalone Usage
+
+Simply install the package with composer and check the [Standalone Usage Example](#standalone-usage-example) below.
+
+```shell
+composer require "behat/mink" "mink/webdriver-classic-driver"
+```
+
+### For Behat/BDD Usage
+
+1. Add all the relevant packages - typically this will be for testing your application, so you'd want to use `--dev`.
+    ```shell
+    composer require --dev "behat/behat" "behat/mink" "friends-of-behat/mink-extension" "mink/webdriver-classic-driver"
+    ```
+2. Configure Behat to enable the Mink Extension and set it to use this driver.
```

Comment:
Would posting a link on https://github.com/FriendsOfBehat/MinkExtension/blob/master/doc/index.rst page, that will explain how to connect this driver specifically to Behat hurt?

// cc @stof .

Reply:
I think it's a good idea.